p4sdk/images/Dockerfile

```diff
@@ -13,8 +13,13 @@ WORKDIR /src
 RUN mkdir -pv /opt/p4 && tar -zxvf p4.tar.gz -C /opt/p4 && rm -f p4.tar.gz
 
 FROM --platform=linux/arm64 alpine:3@sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b
+RUN apk add --no-cache --virtual gawk gettext
 COPY --from=builder /opt/p4 /opt/p4
-WORKDIR /opt/p4
+COPY p4 /root/linux_networking
```

Comment:
I didn't see linux_networking directory created but I'm assuming it is in other stages of the build process.

Reply:
It's get created from Docker COPY command